python/mxnet/operator.py

```diff
@@ -563,7 +563,7 @@ def infer_storage_type_backward(self, in_stype):
             list of aux stypes calculated from in_stype,
             in the same order as declared in list_auxiliary_states.
         """
-        return in_stype, [in_stype[0]]*len(self.list_outputs()), \
+        return in_stype, [in_stype[0]]*len(self.list_arguments()), \
```

Comment:
This looks wrong. Why is there only one argument to infer_storage_type_backward?

Reply:
the argument in_stype is a list of storage types for out_grad, in_data, out_data.

Reply:
That's wrong. There should be multiple lists.